### PR TITLE
memcached-operator/deploy: use specific namespaces in CSV's

### PR DESCRIPTION
**Pull request by estroz**
See https://github.com/operator-framework/getting-started/pull/36#discussion_r275461334.

### CODE REVIEW

File: memcached-operator/deploy/olm-catalog/memcached-operator/0.5.0/memcached-operator.v0.5.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
     alm-examples: '[{"apiVersion":"cache.example.com/v1alpha1","kind":"Memcached","metadata":{"name":"example-memcached"},"spec":{"size":3}}]'
     capabilities: Basic Install
   name: memcached-operator.v0.5.0
-  namespace: placeholder
+  namespace: memcached-operator
```
Comment (hasbro17) [inline memcached-operator/deploy/olm-catalog/memcached-operator/0.5.0/memcached-operator.v0.5.0.clusterserviceversion.yaml]: Can we keep this as `default`.
If we're to use this CSV as the example for the getting-started guide then you'll notice we're running everything in the `default` namespace for that walkthrough.

File: memcached-operator/deploy/olm-catalog/memcached-operator/0.5.0/memcached-operator.v0.5.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ metadata:
     alm-examples: '[{"apiVersion":"cache.example.com/v1alpha1","kind":"Memcached","metadata":{"name":"example-memcached"},"spec":{"size":3}}]'
     capabilities: Basic Install
   name: memcached-operator.v0.5.0
-  namespace: placeholder
+  namespace: default
```
Comment (SamiSousa) [inline memcached-operator/deploy/olm-catalog/memcached-operator/0.5.0/memcached-operator.v0.5.0.clusterserviceversion.yaml]: The namespace will be overwritten by OLM when installed using a catalogsource and subscription, which in my eyes is the only way of installing with OLM. I have no objections with using `default` as a placeholder for CSVs, but the plus side of using `placeholder` is that it discourages using `kubectl apply -f` to add the CSV directly to a cluster

Comment (hasbro17) [reply memcached-operator/deploy/olm-catalog/memcached-operator/0.5.0/memcached-operator.v0.5.0.clusterserviceversion.yaml]: @estroz After second thoughts we'll most likely generate the CSVs as part of the getting-started guide, and we won't be using this CSV directly. Plus the fact that we can't use this CSV right now without a valid image.
https://github.com/operator-framework/getting-started/pull/36#discussion_r277429727

So I would suggest we close this PR and keep `namespace: placeholder` for now.

**Comment by estroz** [comment]
Closing as per https://github.com/operator-framework/operator-sdk-samples/pull/62#discussion_r277449094

**Pull request closed without merging**